cs3api4lab/config/config_manager.py

```diff
@@ -2,14 +2,216 @@
 from jupyter_core.paths import jupyter_config_path
 from notebook.services.config import ConfigManager
 from traitlets.config import LoggingConfigurable
+from traitlets import Unicode, Bool, CInt, Tuple, default
 
 
 class Config(LoggingConfigurable):
-    config = {
-        "auth_token_validity": "3600",
+
```

Comment:
please let me know if this display i.e. the list of variables before constructors is suitable